ENTESB-7329: EncryptionOperationNotPossibleException when calling io.fabric8:type=Fabric/containers() from all fabric containers when wrong encryption key is used.

1984shekhar opened the pull request:
ENTESB-7329: This fix just properly catch the exception and properly log it without printing stacktrace in output of mbean 'io.fabric8:type=Fabric/containers()'.

Commit:
ENTESB-7329: This fix just properly catch the exception and properly log
it without printing stacktrace in output of mbean
'io.fabric8:type=Fabric/containers()'.

tadayosi reviewed (changes requested):
I'd prefer the code formatted more cleanly.

tadayosi commented on fabric/fabric-core/src/main/java/io/fabric8/service/FabricServiceImpl.java:
```diff
@@ -1335,6 +1336,9 @@ public String getValue(String toSubstitute) {
                         return substituteBundleProperty(toSubstitute, bundleContext);
                     }
                 }));
+                }catch(Exception exception){
```
Why don't you just catch `EncryptionOperationNotPossibleException` instead of catcing all?

1984shekhar replied:
tadayosi, you are right I will try to have a better indentation. Secondly I thought that there might be more such scenario where a different exception might be raised.

I will test with catching EncryptionOperationNotPossibleException and than further update.

Commit:
ENTESB-7329: EncryptionOperationNotPossibleException when calling
io.fabric8:type=Fabric/containers() from all fabric containers when
wrong encryption key is used.

Commit:
ENTESB-7329: EncryptionOperationNotPossibleException when calling
io.fabric8:type=Fabric/containers() from all fabric containers when
wrong encryption key is used.

1984shekhar commented:
As this is messed up due to formatting issue, going to create new one
